### PR TITLE
Readme Exercise 3

### PR DESCRIPTION
**Pull request by gara501**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
 # Algorithms
-This repo will have some alghoritms based on different solutions and approaches. 
-Project Structure is simple > Algorithm name > participant > files. 
+This repo will have some alghoritms based on different solutions and approaches.
```
Comment (imagentleman) [inline README.md]: algorithms*

File: README.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
 
-## Exercise 1 - Bubble and lineal search
+## Exercise 1 - Binary and lineal search
+Do 2 different alghoritms using lineal and binary search method.
+Input: Array size (integer)
+Output: Spent time in each alghoritm.
+Rules: The array only will have integers.
 ## Exercise 2 - Three
-The staircase is right-aligned, composed of # symbols and spaces, and has a height and width of n = number inserted. Example, number = 6. 
+The staircase is right-aligned, composed of # symbols and spaces, and has a height and width of n = number inserted. Example, number = 6.
```
Comment (imagentleman) [inline README.md]: Could it be something like:

... height and width of n (a number).

File: README.md [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
 ------------ |
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`#` <br />
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`##`<br />
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`###`<br />
 &nbsp;&nbsp;&nbsp;&nbsp;`####`<br />
 &nbsp;&nbsp;`#####`<br />
 `######`
+
+## Exercise 3 - Matrix Sum
+You will generate a matrix with randomic numbers from 1 to 999999, you need to reverse its rows and columns any number of times and the goal is to maximize the
```
Comment (imagentleman) [inline README.md]: random*

File: README.md [hunk 4]
```diff
@@ -1,16 +1,30 @@
 # Algorithms
-This repo will have some alghoritms based on different solutions and approaches. 
-Project Structure is simple > Algorithm name > participant > files. 
+This repo will have some alghoritms based on different solutions and approaches.
+Project Structure is simple > Algorithm name > participant > files.
 
-## Exercise 1 - Bubble and lineal search
+## Exercise 1 - Binary and lineal search
+Do 2 different alghoritms using lineal and binary search method.
```
Comment (imagentleman) [inline README.md]: algorithms*

**Comment by camorenoa** [comment]
Good Job ~~young~~ old man!

**Pull request merged**